setup.py

```diff
@@ -29,7 +29,7 @@ def run_tests(self):
 # requirements
 install_requirements = ['guessit>=0.9.1', 'babelfish>=0.5.2', 'enzyme>=0.4.1', 'beautifulsoup4>=4.2.0',
                         'requests>=2.0', 'click>=4.0', 'dogpile.cache>=0.5.4', 'stevedore>=1.0.0',
-                        'chardet>=2.3.0', 'pysrt>=1.0.1', 'six>=1.9.0']
+                        'chardet>=2.3.0', 'pysrt>=1.0.1', 'six>=1.9.0', 'rarfile>=2.6']
```

Comment:
Did you test with previous version of rarfiles? The requirements are for api-level compatibility with subliminal.